sway/tree/view.c

```diff
@@ -641,6 +641,14 @@ void view_child_destroy(struct sway_view_child *child) {
 	}
 }
 
+static size_t append_prop(char *buffer, const char *value) {
```

Comment:
Would it be worth moving this to `common/stringop.c` and renaming it to something like `lenient_strcat_size`? I would guess that this is not the only place that could make use of it.

Reply:
My preference is to move it only if there's anything else that can make use of it. So I'd leave it where it is until that happens.

Reply:
Ok. That works